data_ingestion.ipynb

```diff
@@ -22,7 +22,7 @@
         "colab_type": "text"
```

Comment:
<p>It's not clear to me if/how this requires manual intervention in the EE Code Editor. Is this still the case? If so, what currently needs to be done in the Code Editor, and could we use the Python API instead?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/tnc-br/ddf-isoscapes/pull/197/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='13'/>

Reply:
It does not require manual intervention in the EE Code Editor. But it does require waiting for EE to export the raster to the cache folder in Drive. You can check the progress of that task in EE Code Editor, but it's not required.